h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeMojoModel.java

```diff
@@ -57,6 +58,14 @@
      * GLM's beta used for calibrating output probabilities using Platt Scaling.
      */
     protected double[] _calib_glm_beta;
+    
+    protected String _genmodel_encoding;
```

Comment:
I think these changes are not backwards compatible so you will need to bump the version of all affected MOJOs and update MOJOLand

Reply:
Good point, we need to prevent users from running with an old runtime with a newer mojo.